src/SFA.DAS.LevyTransferMatching.Web/Orchestrators/PledgeOrchestrator.cs

```diff
@@ -38,46 +52,77 @@ public async Task<CreateViewModel> GetCreateViewModel(CreateRequest request)
                 Sectors = cacheItem.Sectors,
                 JobRoles = cacheItem.JobRoles,
                 Levels = cacheItem.Levels,
+                LevelOptions = levels,
+                SectorOptions = sectors,
+                JobRoleOptions = jobRoles
             };
         }
 
         public async Task<AmountViewModel> GetAmountViewModel(AmountRequest request)
         {
             var cacheItem = await RetrievePledgeCacheItem(request.CacheKey);
+            var remainingTransferAllowance = await _accountsService.GetRemainingTransferAllowance(request.EncodedAccountId);
         
             return new AmountViewModel
             {
                 EncodedAccountId = request.EncodedAccountId,
                 CacheKey = request.CacheKey,
                 Amount = cacheItem.Amount.ToString(),
+                RemainingTransferAllowance = remainingTransferAllowance.ToString("N0"),
                 IsNamePublic = cacheItem.IsNamePublic
             };
         }
 
         public async Task<SectorViewModel> GetSectorViewModel(SectorRequest request)
         {
             var cacheItem = await RetrievePledgeCacheItem(request.CacheKey);
+            var sectors = await _tagService.GetSectors();
```

Comment:
Similar to a previous comment, as these two awaits are not related they could be awaited together using WhenAll
```suggestion
            var cacheItemTask = RetrievePledgeCacheItem(request.CacheKey);
            var sectorsTask = _tagService.GetSectors();
            await Task.WhenAll(cacheItemTask, sectorsTask);
```